client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-panel.scss

```diff
@@ -0,0 +1,17 @@
+.pattern-list-panel__wrapper {
+	position: absolute;
+	z-index: 1;
+	top: -60px;
+	left: 0;
```

Comment:
We have to handle the right-to-left direction. It would be better to do it in the follow-up PR as there are some issues there 😓

Reply:
Okay! https://github.com/Automattic/wp-calypso/issues/74320